src/aws-ecr/orb.yml

```diff
@@ -0,0 +1,143 @@
+version: 2.1
+
+description: |
+  An orb for working with Amazon Elastic Container Registry (ECR)
+
+  The executors within this Orb provide run-time environments that contain the 
+  necessary dependencies for executing the jobs and commands in this orb, 
+  except for the following required dependencies that must be configured in 
+  CircleCI in order to use this orb - Docker, environment variables for AWS 
+  login ($AWS_ACCESS_KEY_ID, $AWS_SECRET_ACCESS_KEY)
+
+  The source for this Orb can be found here:
+  https://github.com/CircleCI-Public/circleci-orbs/tree/master/src/aws-ecr
+
+examples:
+  Using job to build and push image to Amazon ECR:
```

Comment:
We opted for `examples` keys using the same format as e.g. workflow and job names, so only alphanumerical characters, `-`, and `_` are allowed. I suggest something like `simple_build_and_push`, and then incorporating `Using job to build and push image to Amazon ECR` into the `description` field.

The `examples` feature is already live, so you can validate your orb.yml with `circleci orb validate` CLI command.